schemas/mozilla_nimbus_schemas/tests/jetstream/test_population_sizing.py

```diff
@@ -13,15 +13,14 @@ def test_parse_population_sizing(self):
         """Test against known good data."""
         sizing_test_data = """
         {
-            "firefox_desktop:release:['EN-US']:US:108": {
+            "firefox_desktop:release:['EN-US']:US": {
```

Comment:
Curious as to why you removed this?

Reply:
This is part of what was removed from the formatting of the output that we're ingesting, so I wanted to represent it in the test data. It's hard to represent in the schema itself because it's a concatenation of values, and having an enum of all possible combinations would be huge.

Reply:
theoretically could describe the format with a regex pattern (like https://json-schema.org/understanding-json-schema/reference/regular_expressions.html) but I dont know how to express that in pydandic and its probably overkill

Reply:
Hm that's an interesting thought, but yea I would agree that it's probably overkill here.